fcrepo-webapp/src/test/java/org/fcrepo/integration/FedoraHtmlResponsesIT.java

```diff
@@ -134,14 +143,38 @@ public void testCreateNewNodeWithGeneratedId() throws IOException {
         final HtmlPage page = webClient.getPage(serverAddress);
         final HtmlForm form = (HtmlForm)page.getElementById("action_create");
         final HtmlSelect type = (HtmlSelect)page.getElementById("new_mixin");
-        type.getOptionByValue("container").setSelected(true);
+        type.getOptionByValue("basic container").setSelected(true);
         final HtmlButton button = form.getFirstByXPath("button");
         button.click();
 
         final HtmlPage page1 = javascriptlessWebClient.getPage(serverAddress);
         assertTrue("Didn't see new information in page!", !page1.asText().equals(page.asText()));
     }
 
+    @Test
+    public void testCreateNewBasicContainer() throws IOException {
+        final HtmlPage newPage = createAndVerifyObjectWithIdFromRootPage(newPid(), "basic container");
+        System.err.println(newPage.asText());
```

Comment:
Please remove this `System.err` statement.

Reply:
Just some debugging cruft...